clients/filter/filter.go

```diff
@@ -143,7 +156,10 @@ func (c *Client) CreateJob(datasetFilterID string) (string, error) {
 		return "", err
 	}
 
-	resp, err := c.cli.Post(c.url+"/filters", "application/json", bytes.NewBuffer(b))
+	uri := c.url + "/filters"
+	clientlog.Do(fmt.Sprintf("attemping to create filter job with instance id: %s", datasetFilterID), service, uri, "POST")
```

Comment:
if Do accepted an optional `log.Data` param, for post/put requests the relevant fields could be passed through in a structured way?